ci-operator/step-registry/upi/libvirt/install/heterogeneous/upi-libvirt-install-heterogeneous-commands.sh

```diff
@@ -0,0 +1,346 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ "${ADDITIONAL_WORKER_ARCHITECTURE}" != "x86_64" ]; then
```

Comment:
I think I'd appreciate more coverage in the long run, but really this can be something fixed when and if a problem comes up where someone tries with `ADDITIONAL_WORKER_ARCHITECTURE = 'amd64'` or something.